hasher-matcher-actioner/hmalib/models.py

```diff
@@ -74,14 +78,34 @@ def to_sqs_message(self) -> dict:
             "key": self.content_key,
         }
 
+    @classmethod
+    def get_from_content_key(
+        cls, table: Table, content_key: str
+    ) -> t.Union["PipelinePDQHashRecord", None]:
+        items = HashRecordQuery.from_content_key(
+            table,
+            cls.get_dynamodb_content_key(content_key),
+            cls.get_dynamodb_type_key(cls.SIGNAL_TYPE),
+        )
+        records = []
+        for item in items():
+            records.append(
+                PipelinePDQHashRecord(
+                    item["PK"],
+                    item["ContentHash"],
+                    item["UpdatedAt"],
+                    item["Quality"],
+                )
+            )
+        return None if not len(records) else records[0]
```

Comment:
```suggestion
        return None if not records else records[0]
```

If you want to be extra silly you can do:
```
return (records or [None])[0]
```